workos/portal.py

```diff
@@ -0,0 +1,44 @@
+import workos
+from workos.utils.request import RequestHelper, REQUEST_METHOD_GET
+from workos.utils.validation import PORTAL_MODULE, validate_settings
+
+RESPONSE_LIMIT = 10
+
+
+class Portal(object):
+    @validate_settings(PORTAL_MODULE)
+    def __init__(self):
+        pass
+
+    @property
+    def request_helper(self):
+        if not getattr(self, "_request_helper", None):
+            self._request_helper = RequestHelper()
+        return self._request_helper
+
+    def list_organizations(
+        self, domain=None, limit=RESPONSE_LIMIT, before=None, after=None
```

Comment:
Looks like the params expects on API are domains (plural not singular). Does this work as is?

Reply:
This is a great catch @henrylamchan looks like my original spec was wrong and I tracked this bug across multiple SDKs. I'll fix here and give a fix to Node and Ruby before releasing them. Good eye!